WasabiWords.lua

```diff
@@ -609,7 +664,16 @@ wasabi_words:add_button("ShinyWasabi", function() --Original script by ShinyWasa
         stats.set_int("MPX_SALV23_GEN_BS", -1) -- polgauntlet trade price
         stats.set_int("MPX_SALV23_SCOPE_BS", -1) -- police5 trade price
         stats.set_int("MPX_MOST_TIME_ON_3_PLUS_STARS", 300000) -- police4 trade price
-        stats.set_int("MPX_LOWRIDER_FLOW_COMPLETE", 1)
+        stats.set_int('MPX_LOWRIDER_FLOW_COMPLETE', 3)
+        stats.set_int('MPX_LOW_FLOW_CURRENT_PROG', 9)
+        stats.set_int('MPX_LOW_FLOW_CURRENT_CALL', 9)
+        stats.set_bool('MPX_HAS_WATCHED_BENNY_CUTSCE', true)
+        stats.set_bool('MPX_LOW_FLOW_CS_DRV_SEEN', true)
+        stats.set_bool('MPX_LOW_FLOW_CS_TRA_SEEN', true)
+        stats.set_bool('MPX_LOW_FLOW_CS_FUN_SEEN', true)
+        stats.set_bool('MPX_LOW_FLOW_CS_PHO_SEEN', true)
+        stats.set_bool('MPX_LOW_FLOW_CS_FIN_SEEN', true)
+        stats.set_bool('MPX_LOW_BEN_INTRO_CS_SEEN', true)
```

Comment:
What is this for? I don't see these checked in the scripts.

They are bool stats 174 -> 180. Remove them if not required.